packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -50,64 +48,61 @@ export class OrgList implements vscode.Disposable {
     }
   }
 
-  public async getAuthInfoObjects() {
-    const authFilesArray = await AuthInfo.listAllAuthorizations().catch(
+  public async getOrgAuthorizations(): Promise<OrgAuthorization[] | null> {
```

Comment:
dumb q: was the return type inferred before and we just go explicit about it

Reply:
Yep.  dumb q: do we _always_ want to have explicit return types on functions?  Up until now I haven't been too concerned about it since TS seems to know about the return types even when they are not always explicit...

Reply:
Letting typescript do it's job and inferring the return type is usually fine.  The one exception is if we need to narrow the return type verse the inferred value. my 2c